lib/secure_variable.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+class SecureVariable:
+    def __init__(self, value):
+        self._value = value
+
+    def __repr__(self):
+        return '****OBFUSCATED****'
+
+    def __str__(self):
+        return self.__repr__()
+
+    def get_value(self):
+        return self._value
+
+class SecureVariableEncoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, SecureVariable):
+            return repr(o)
+        return super().default(o)
+
+
+if __name__ == '__main__':
```

Comment:
I would put this in a test

Reply:
Whenever I do these mini codes I see them more as a tooling and manual evaluation if in question. I would keep it in for now and remove later if it becomes overwhelming